{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "8f6c3916_ad62e5fb",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 14
      },
      "lineNbr": 0,
      "author": {
        "id": 1271173
      },
      "writtenOn": "2021-01-05T01:31:48Z",
      "side": 1,
      "message": "Retry CQ due to same failure pattern from: crbug.com/1162595.",
      "revId": "80b08ee557058711dd1016c3dd0cd7441b73b708",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}